src/components/TouchButton.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import { motion } from 'framer-motion';
+import { ComponentProps } from 'react';
+
+type Props = ComponentProps<typeof motion.button>;
+
+export function TouchButton({ children, ...restProps }: Props) {
+  return (
+    <motion.button
+      initial="idle"
+      whileTap="touch"
+      variants={{ idle: { scale: 1 }, touch: { scale: 0.96 } }}
+      transition={{ duration: 0.1 }}
+      {...restProps}>
+      {children}
+    </motion.button>
+  );
```

Comment:
```suggestion
<motion.button
  initial="idle"
  whileTap="touch"
  variants={{ idle: { scale: 1 }, touch: { scale: 0.96 } }}
  transition={{ duration: 0.1 }}
  {...restProps}
/>;
```

이러한 사용은 어떻게 생각하시나요?? 
`children`을 `...restProps`에 포함되게 한 경우입니다 !

Reply:
지금 children은 별도의 prop으로 빠져있어요! 이거 린팅룰이 가독성이 좀 안좋네요.. ㅋㅋㅋㅋㅋ